org.gnome.Games.LibretroPlugin.PicoDrive.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "org.gnome.Games.LibretroPlugin.PicoDrive",
+    "runtime": "org.gnome.Games",
+    "runtime-version": "3.24",
```

Comment:
The merged version of Games has updated to the 3.26 version, I assume a mismatch could cause issues?